MC/bin/o2dpg_qc_finalization_workflow.py

```diff
@@ -49,6 +49,7 @@ def add_QC_finalization(taskName, qcConfigPath, needs=[]):
   add_QC_finalization('mftDigitsQC', 'json://${O2DPG_ROOT}/MC/config/QC/json/qc-mft-digit.json')
   add_QC_finalization('mftClustersQC', 'json://${O2DPG_ROOT}/MC/config/QC/json/qc-mft-cluster.json')
   add_QC_finalization('mftAsyncQC', 'json://${O2DPG_ROOT}/MC/config/QC/json/qc-mft-async.json')
+  add_QC_finalization('emcAsyncQC', 'json://${O2DPG_ROOT}/MC/config/QC/json/qc-emc-async.json')
```

Comment:
MFT used "async" in their file name, because their QC task is called `QcMFTAsync`. Please use something alongside of `emcDigitsQC` and `qc-emc-digits` to stay in line with the existing convention.

Reply:
OK, will rename the config to `emc-digits-task` and the taskname to emcDigitsQc.